internal/ethapi/transaction_args.go

```diff
@@ -537,9 +582,9 @@ func (args *TransactionArgs) ToTransaction() *types.Transaction {
 			Sender:                      args.Sender,
 			AuthorizationData:           *args.AuthorizationData,
 			Paymaster:                   args.Paymaster,
-			PaymasterData:               *args.PaymasterData,
+			PaymasterData:               toByte(args.PaymasterData),
```

Comment:
Just curious, what was the intention of this?

Reply:
Code to prevent nil pointer reference from occurring